webpack/ForemanTasks/Components/TaskDetails/TaskDetailsReducer.js

```diff
@@ -1,42 +1,20 @@
 import Immutable from 'seamless-immutable';
-import { STATUS } from 'foremanReact/constants';
 import {
-  FOREMAN_TASK_DETAILS_FETCH_TASK_REQUEST,
-  FOREMAN_TASK_DETAILS_FETCH_TASK_SUCCESS,
   FOREMAN_TASK_DETAILS_STOP_POLLING,
   FOREMAN_TASK_DETAILS_START_POLLING,
-  FOREMAN_TASK_DETAILS_FETCH_TASK_FAILURE,
+  FOREMAN_TASK_DETAILS_SUCCESS,
 } from './TaskDetailsConstants';
 
 const initialState = Immutable({ isData: false });
 
-export default (state = initialState, action) => {
-  const { type, payload } = action;
-  let { taskReload, timeoutId } = state;
-
+export default (state = initialState, { type }) => {
   switch (type) {
-    case FOREMAN_TASK_DETAILS_FETCH_TASK_REQUEST:
-      return state.set('status', STATUS.PENDING);
-    case FOREMAN_TASK_DETAILS_FETCH_TASK_SUCCESS:
-      if (payload.state === 'stopped') {
-        clearTimeout(state.timeoutId);
-        taskReload = false;
-        timeoutId = null;
-      }
-      return state.merge({
-        status: STATUS.RESOLVED,
-        isData: true,
-        ...payload,
-        taskReload,
-        timeoutId,
-      });
-    case FOREMAN_TASK_DETAILS_FETCH_TASK_FAILURE:
-      return state.merge({ status: STATUS.ERROR, error: payload });
+    case FOREMAN_TASK_DETAILS_SUCCESS:
+      return state.set('isData', true);
     case FOREMAN_TASK_DETAILS_STOP_POLLING:
-      return state.merge({ taskReload: false, timeoutId: null });
+      return state.set('taskReload', false);
     case FOREMAN_TASK_DETAILS_START_POLLING:
-      clearTimeout(state.timeoutId);
-      return state.merge({ taskReload: true, timeoutId: payload.timeoutId });
+      return state.set('taskReload', true);
```

Comment:
why do you need to manage a local state? 
to check if the interval exist or not you can use `selectDoesIntervalExist(key)`

Reply:
More lines to delete :tada: